src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,671 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import * as _ from 'lodash'
+
+import { Extendable, ComponentEventHandler } from '../../../types/utils'
+import { ComponentSlotStylesInput, ComponentVariablesInput } from '../../themes/types'
+import Downshift, {
+  DownshiftState,
+  StateChangeOptions,
+  A11yStatusMessageOptions,
+  GetMenuPropsOptions,
+  GetPropsCommonOptions,
+  GetInputPropsOptions,
+  GetToggleButtonPropsOptions,
+  GetItemPropsOptions,
+} from 'downshift'
+import Label from '../Label/Label'
+import { AutoControlledComponent, RenderResultConfig } from '../../lib'
+import Input from '../Input/Input'
+import keyboardKey from 'keyboard-key'
+import List from '../List/List'
+import Text from '../Text/Text'
+import Image from '../Image/Image'
+import ListItem from '../List/ListItem'
+import Icon from '../Icon/Icon'
+import { commonUIComponentPropTypes } from '../../lib/commonPropTypes'
+import Ref from '../Ref/Ref'
+
+// To be replaced when Downshift will add highlightedItem in their interface.
+export interface DownshiftA11yStatusMessageOptions<Item> extends A11yStatusMessageOptions<Item> {
+  highlightedItem: Item
+}
+
+export interface DropdownProps {
```

Comment:
You should extends the props with the UIComponentProps. I don't see here any of the common props (styles, variables, as etc.)

Reply:
`export interface DropdownProps extends UIComponentProps<any, any> {`